exporter/mezmoexporter/factory.go

```diff
@@ -46,17 +47,22 @@ func createDefaultConfig() config.Exporter {
 }
 
 // Create a log exporter for exporting to Mezmo
-func createLogsExporter(ctx context.Context, settings component.ExporterCreateSettings, exporter config.Exporter) (component.LogsExporter, error) {
-	if exporter == nil {
+func createLogsExporter(ctx context.Context, settings component.ExporterCreateSettings, exporterConfig config.Exporter) (component.LogsExporter, error) {
+	log := settings.Logger
+	log.Info("creating Mezmo logs exporter")
+
+	if exporterConfig == nil {
+		log.Error("cannot create exporter with empty configuration")
 		return nil, errors.New("nil config")
 	}
-	expCfg := exporter.(*Config)
+	expCfg := exporterConfig.(*Config)
 
 	if err := expCfg.Validate(); err != nil {
+		log.Error("cannot create exporter with invalid configuration", zap.Error(err))
```

Comment:
We do print errors on the "caller" of the create if error happens.

Reply:
I have updated the PR by removing the superfluous logs.